### PR TITLE
Get Ceph from distro repository (debian-based)

### PR DESCRIPTION
**Pull request by HanXHX**
Hi,

Some people prefers install package from their distro. This PR provides this feature as optional (default stays from ceph).

Cheers,

Emilien

### CODE REVIEW

**Comment by leseb** [comment]
Hi @HanXHX, thanks for pushing this.
I like the idea, could you do the same for redhat based systems as well? It shouldn't be too much work.
One more thing, don't forget to reflect in `group_vars/all.sample` all the changes you introduce in `defaults/main.yml`.

**Comment by HanXHX** [comment]
Hi @leseb,

I add a new commit to reflect all changes. I'm not an expert about RedHat systems... 

Cheers!

**Comment by HanXHX** [comment]
Hi @leseb,

Can you merge please?

**Comment by leseb** [comment]
@HanXHX let me have a final look :)

File: roles/ceph-common/tasks/installs/install_on_debian.yml [hunk 1]
```diff
@@ -7,49 +7,15 @@
...
-  changed_when: false
-  when: ceph_dev
-
-- name: add inktank ceph enterprise repository
-  apt_repository: >
-    repo="deb file://{{ ceph_stable_ice_temp_path }} {{ ansible_lsb.codename }} main"
-    state=present
-  changed_when: false
-  when: ceph_stable_ice
+- name: configure Ceph APT repository
```
Comment (leseb) [inline roles/ceph-common/tasks/installs/install_on_debian.yml]: could just remove the capital letters? so we keep this consistent with the rest of the playbook.

**Comment by leseb** [comment]
Thanks @HanXHX

**Pull request merged**